lib/ood_core/job/adapters/linux_host/templates/script_wrapper.erb.sh

```diff
@@ -14,6 +14,7 @@ if [ -z "$hostname" ]; then
 fi
 
 echo $hostname
+echo ""
```

Comment:
I think we'd want to echo before, not after right? That way we ensure that `echo $hostname` is on it's own line.